class/defaults.yml

```diff
@@ -16,3 +16,31 @@ parameters:
 
     helm_values:
       enableCertManager: true
+
+    monitoring_enabled: true
```

Comment:
New component parameters should be documented in `docs/modules/ROOT/pages/references/parameters.adoc`